chatsky/messengers/http_interface.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+from typing import Optional
+
+import dotenv
+import uvicorn
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+from chatsky.core import Message
+from chatsky.messengers.common import MessengerInterface
+
+dotenv.load_dotenv()
+
+HTTP_INTERFACE_PORT = int(os.getenv("HTTP_INTERFACE_PORT", 8020))
```

Comment:
Why get port via env and why load env variables from dotenv?
IMO port should be set via `__init__`.